bdk-ffi/src/bitcoin.rs

```diff
@@ -548,6 +559,15 @@ impl Psbt {
         }
     }
 
+    /// Write the `Psbt` to a file. Note that the file must not yet exist.
+    pub(crate) fn write_to_file(&self, path: String) -> Result<(), PsbtError> {
+        let file = File::create(path)?;
```

Comment:
Wondering your thoughts on this: doc comment says "Note that the file must not yet exist." but `File::create()` "will truncate it if it does" (exist) per [docs](https://doc.rust-lang.org/std/fs/struct.File.html#method.create). Should this use [`File::create_new()`](https://doc.rust-lang.org/std/fs/struct.File.html#method.create_new) instead to prevent overwrites? Or I could be misunderstanding workflow/behavior too.